serverless/src/cmrGranuleSearch.js

```diff
@@ -0,0 +1,78 @@
+import request from 'request-promise'
+import jwt from 'jsonwebtoken'
+import fs from 'fs'
+import { stringify as qsStringify } from 'qs'
+import { pick } from './util'
+
+const config = JSON.parse(fs.readFileSync('config.json'))
```

Comment:
I'm not seeing this file in the PR? But.. we don't want client id in the repo. We should move this to using something secure. This is a challenge with local development, we'll need to figure that out.